src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -729,6 +774,7 @@ def test_sql_db_long_term_retention(
         })
 
         # test update long term retention on live database
+        print('Creating MI...\n')
```

Comment:
is it required?

Reply:
just fur debug. will remove it in next commit.

Reply:
removed